resource_arm_application_gateway: add disabled_rule_group to waf_configuration (#3394)
terraform-providers/terraform-provider-azurerm: website/docs/r/application_gateway.html.markdown

* `rule_set_version` - (Required) The Version of the Rule Set used for this Web Application Firewall.

> * `disabled_rule_group` - (Optional) one or many `disabled_rule_group` blocks as defined below.

Reviewer: We should keep this message consistent with the other attributes of a similar nature
```suggestion
* `disabled_rule_group` - (Optional) One or more `disabled_rule_group` blocks as defined below.
```

---

A `disabled_rule_group` block supports the following:

* `rule_group_name` - (Required) The rule group where specific rules should be disabled. Accepted values are:  `crs_20_protocol_violations`, `crs_21_protocol_anomalies`, `crs_23_request_limits`, `crs_30_http_policy`, `crs_35_bad_robots`, `crs_40_generic_attacks`, `crs_41_sql_injection_attacks`, `crs_41_xss_attacks`, `crs_42_tight_security`, `crs_45_trojans`, `REQUEST-911-METHOD-ENFORCEMENT`, `REQUEST-913-SCANNER-DETECTION`, `REQUEST-920-PROTOCOL-ENFORCEMENT`, `REQUEST-921-PROTOCOL-ATTACK`, `REQUEST-930-APPLICATION-ATTACK-LFI`, `REQUEST-931-APPLICATION-ATTACK-RFI`, `REQUEST-932-APPLICATION-ATTACK-RCE`, `REQUEST-933-APPLICATION-ATTACK-PHP`, `REQUEST-941-APPLICATION-ATTACK-XSS`, `REQUEST-942-APPLICATION-ATTACK-SQLI`, `REQUEST-943-APPLICATION-ATTACK-SESSION-FIXATION`

> * `rules` - (Required) A list of rule numbers which should be disabled in that group. Example: [921110, 921151, 921180] if group is set to `REQUEST-921-PROTOCOL-ATTACK`

Reviewer 2: Need to update here too, something like:
```suggestion
* `rules` - (Optional) A list of rule numbers which should be disabled in that group. Example: [921110, 921151, 921180] if group is set to `REQUEST-921-PROTOCOL-ATTACK`. Disables all rules in the group if `rules` is not specified.
```

> * `rules` - (Optional) A list of rule numbers which should be disabled in that group. Example: [921110, 921151, 921180] if group is set to `REQUEST-921-PROTOCOL-ATTACK`. Disables all rules in the group if `rules` is not specified.

Reviewer: An example usage isn't recommended in descriptions for attributes.
```suggestion
* `rules` - (Optional) A list of rules which should be disabled in that group.  Disables all rules in the specified group if `rules` is not specified.
```
